### ISSUE TITLE
Clean up data pre-processing scripts

### ISSUE THREAD

**Issue by raspstephan** [raspstephan/CBRAIN-Keras-Diagnostics#2]
Currently, the data preprocessing is handled by the preprocess_aqua.py function followed by shuffle_ds.py to randomize the training dataset. To create a new training + validation set, three commands are required. Much of this is redundant and should be easier, so that only one config script is needed for all of these steps.

Additionally, some of the functionality in preprocess_aqua is deprecated.
